CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.20.0 / unreleased
+
+* [ENHANCEMENT] TSDB: WAL compression is enabled by default.
```

Comment:
Should it be a `CHANGE`? Compression was already available as an enhancement.

Reply:
Yes.

Reply:
This is not a CHANGE, this doesn't break anything. It's an enhancement.

Reply:
With that I realised, we have not been updating CHANGELOG in every PR, rather we do it during the release process. Is this something to be done in other PRs too?

Reply:
no that would be merge issues every time

Reply:
I personally like to have CHANGELOG entries in my PRs. We do this for the node_exporter and it works out pretty well. Yes, we get some occasional merge issues, but not every time.

It helps to have empty entries like this:
```
* [CHANGE]
```

This helps git figure out how to merge changes in.

Reply:
@brian-brazil It does break rollback. But since we don't guarantee rollbacks, I think ENHANCEMENT is also fine.

Reply:
>  Is this something to be done in other PRs too?

It's a matter of personal preference.